scripts/wsrep_sst_common.sh

```diff
@@ -1004,7 +1005,7 @@ function exec_sql() {
 #
 # Parameters:
 #   None
-#
+# 
```

Comment:
Please remove the whitespace at the end of line

Reply:
done